WordPress/Classes/Utility/Editor/EditorFactory.swift

```diff
@@ -27,6 +27,7 @@ class EditorFactory {
             gutenbergSettings.setGutenbergEnabled(true, for: post.blog)
             gutenbergSettings.postSettingsToRemote(for: post.blog)
             gutenbergVC.shouldPresentInformativeDialog = true
+            gutenbergSettings.willShowDialog(for: post.blog)
```

Comment:
I don't understand why we're adding this, since `setGutenbergEnabled` will also set the flag

Reply:
Answered here: https://github.com/wordpress-mobile/WordPress-iOS/pull/12235#issuecomment-517660414 to have more space :)